tools/nomad-generate/structs.copy.tmpl

```diff
@@ -0,0 +1,82 @@
+package structs
```

Comment:
To make Go happy I think we should emit a comment matching: 

```
^// Code generated .* DO NOT EDIT\.$
```

https://golang.org/pkg/cmd/go/internal/generate/